aioslacker/__init__.py

```diff
@@ -29,12 +30,15 @@ def __init__(
 
         self.loop = loop
 
-        self.session = aiohttp.ClientSession(
-            connector=aiohttp.TCPConnector(
-                use_dns_cache=False,
-                loop=self.loop,
-            ),
-        )
+        if not isinstance(session, aiohttp.ClientSession):
```

Comment:
This check makes no sense, as well can hard to debug, just remove this check, please.

Reply:
if session is None, then create default session